osu.Game/Overlays/IDialogOverlay.cs

```diff
@@ -0,0 +1,31 @@
+#nullable enable
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
```

Comment:
```suggestion
// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
// See the LICENCE file in the repository root for full licence text.

#nullable enable
```

(see CI, codefilesanity is getting confused by the nullability annotation coming first)